test/aqua/test_shor.py

```diff
@@ -66,6 +67,43 @@ def test_shor_bad_input(self, n_v):
         with self.assertRaises(ValueError):
             Shor(n_v)
 
+    @idata([[2, 15, 8]])
+    @unpack
+    def test_shor_modinv(self, a_v, m_v, expected):
+        """ shor modular inverse test """
+        modinv = Shor.modinv(a_v, m_v)
+        self.assertTrue(modinv == expected)
+
+    @idata([[3, "0011"],
+            [5, "0101"]])
+    @unpack
+    def test_phi_add_gate(self, addition_magnitude, expected_state):
```

Comment:
Unit tests are normally limited to public methods as its the public contract with the entity that is supposed to be tested. Private methods are part of the internal implementation that refactoring could change while maintaining the public interface behavior and the test should run still after refactoring. If it also checks internal methods then it can fail if these are changed. Paths through these internal methods should end up being exercised via testing the public interface. As such it is best practice to not be explicitly testing private methods.

Reply:
Thanks for the explanation @woodsp-ibm - will remove it